parm/land/prep/incr.nml.j2

```diff
@@ -0,0 +1,9 @@
+&noahmp_snow
+ date_str = "{{ current_cycle | to_YMD }}",
+ hour_str = "{{ current_cycle.strftime('%H') }}",
+ res = {{ CASE[1:] }},
+ frac_grid = "{{ FRACGRID }}",
```

Comment:
I would like to see a change in the program `apply_incr.exe`.
`FRACGRID` is expected to be a 3 letter character e.g. `YES`.
The code conditions on `YES` (all uppercase), all other exceptions are treated as `NO` or `.false`.
This means, `yes`, `Yes` are treated as `.false`.
Besides, `FRACGRID` can be inferred based on the contents of `otype` file being read, so ideally, the code can internally resolve if it is processing fractional grid or not.
Thoughts?

Reply:
This will require the help and actions from @barlage and @ClaraDraper-NOAA to make such changes.

Reply:
I am happy to open a PR if they are willing to accept.  It is trivial to make this correction and update.

Reply:
I think we should double check with Mike and Clara and then have @jiaruidong2017 take the lead on making the changes, as to not put too much burden on @aerorahul